code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -72,17 +26,13 @@ BONUS
 	symptom_delay_min = 7
 	symptom_delay_max = 14
 
-/datum/symptom/revitiligo/Activate(datum/disease/advance/A)
+/datum/symptom/polyvitiligo/Activate(datum/disease/advance/A)
 	if(!..())
 		return
 	var/mob/living/M = A.affected_mob
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(H.skin_tone == "african2")
-			return
-		switch(A.stage)
-			if(5)
-				H.skin_tone = "african2"
-				H.update_body(0)
-			else
-				H.visible_message("<span class='warning'>[H] looks a bit dark...</span>", "<span class='notice'>Your skin suddenly appears darker...</span>")
+	switch(A.stage)
+		if(5)
+			M.reagents.add_reagent(/datum/reagent/colorful_reagent, 30)
```

Comment:
This is actually fairly dangerous. You might overload someone with 1000 units of colorful reagents and then they won't be able to take other chems

Reply:
no, should probably cap it at something